lib/config/migration.ts

```diff
@@ -195,6 +195,9 @@ export function migrateConfig(
         } else if (val === false) {
           migratedConfig.trustLevel = 'low';
         }
+      } else if (key === 'ignoreNpmrcFile') {
+        delete migratedConfig.ignoreNpmrcFile;
+        migratedConfig.npmrc ||= '';
```

Comment:
```suggestion
        migratedConfig.npmrc ??= '';
```
Only assing if not `null` or `undefined`